dashboard/test/controllers/api/v1/section_instructors_controller_test.rb

```diff
@@ -165,6 +165,48 @@ class Api::V1::SectionInstructorsControllerTest < ActionController::TestCase
     put :decline, params: {id: si.id}
     assert_response :forbidden
   end
+
+  test 'logged-out user cannot check email addresses' do
+    get :check, params: {email: @teacher2.email}
+    assert_response :forbidden
+  end
+
+  test 'teacher can check other teacher for eligibility' do
+    sign_in @teacher
+    get :check, params: {email: @teacher2.email}
+    assert_response :success
+  end
+
+  test 'teacher gets error checking own email address' do
+    sign_in @teacher
+    get :check, params: {email: @teacher.email}
+    assert_response :bad_request
+  end
+
+  test 'teacher gets error checking non-user email address' do
```

Comment:
Test case for checking gets error checking a student email address?

Reply:
We don't actually store student email addresses. https://blog.code.org/post/147756946588/codeorgs-new-login-approach-to-student-privacy